lib/averages.js

```diff
@@ -0,0 +1,43 @@
+function total(array){
+  let sum = 0;
+  array.forEach(function(value){
+  	sum = sum + value;
+  });
+  return sum;
+}
+let array = [1,2,3,4,5,3];
+console.log(total(array));
```

Comment:
you'll want to remove console logging and testing from your files before you push your code up. ideally, this will be done in your test files 👍